phpunit.xml.dist

```diff
@@ -3,7 +3,7 @@
 <!-- http://phpunit.de/manual/current/en/appendixes.configuration.html -->
 <phpunit
         colors="true"
-        bootstrap="vendor/autoload.php">
+        bootstrap="phpunit.php">
```

Comment:
Let's move this file to the `tests` folder. It's not necessary to locate them in root dir.

And I prefer to rename it to `bootstrap.php`. Is it sounds good?